### PR TITLE
Outsider RD consoles that cant sync to station: Also atmos fixes

### PR DESCRIPTION
**Pull request by tacoguy7765093**
## About The Pull Request
This will replace all outsider RND consoles with a 'public' version that has no access to restricted NT research servers.
This also fixes an atmos issue with the ALL the outsider pods so they have air in them again!
## Changelog
:cl:
add: Outsider RD consoles replace their Core RD consoles
maptweak: Science pods oxygen fixes, no more choking in the pods!
/:cl:

### CODE REVIEW

File: modular_chomp/code/game/objects/items/weapons/circutboards/computer/research.dm [hunk 1]
```diff
@@ -0,0 +1,7 @@
+//Public RND console for outsider pod use
+/obj/item/weapon/circuitboard/rdconsole/public
+	name = T_BOARD("Outsider R&D control console")
+	build_path = /obj/machinery/computer/rdconsole/public
+
+/obj/item/weapon/circuitboard/rdconsole/attackby(obj/item/I as obj, mob/user as mob) //It has access change interactions with screwdriver for the main board, you cant change them here.
```
Comment (Kashargul) [inline modular_chomp/code/game/objects/items/weapons/circutboards/computer/research.dm:6]: All rdconsole boards? Not only the public subtype?

Comment (tacoguy7765093) [reply modular_chomp/code/game/objects/items/weapons/circutboards/computer/research.dm:6]: Oops, missed that, will fix soon

**Comment by Kashargul** [comment]
You need to instance a new board first

**Comment by tacoguy7765093** [comment]
Already instanced? Also CI finding some unrelated RNG runtime, might need to be kicked.

**Pull request merged**